keywords.robot

```diff
@@ -1480,7 +1480,10 @@ Get Cbmem From Cloud
     [Documentation]    Download cbmem from the cloud.
     ${cbmem_path}=    Set Variable    /usr/local/bin/cbmem
     ${out_sha256sum}=    Execute Command In Terminal    sha256sum ${cbmem_path}
-    ${sha256}=    Set Variable    ${out_sha256sum.split()}[0]
+    ${sha256}=    Set Variable    null
```

Comment:
I think it would be more clear if this line was moved to ```ELSE``` part of the ```IF``` below.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/compare/47fba484de7fdf64c98e2f4f7f383a2ea86b400b..623c8aad979a5274ae777c8a1f27038655e3d869